modules/rosa-sts-byo-oidc-options.adoc

```diff
@@ -30,12 +29,12 @@ $ rosa create oidc-config --raw-files
 ----
 
 [discrete]
-[id="rosa-oidc-mode_{context}"]
+[id="rosa-sts-byo-oidc-mode_{context}"]
 == mode
 
-Allows you to specify the mode to create your OIDC configuration. With the `manual` option, you receive AWS commands that setup the OIDC configuration within an S3 bucket. This option stores the private key in the Secrets Manager. With the `manual` option, the OIDC Endpoint URL is the URL for the S3 bucket. You must retrieve the Secrets Manager ARN to register the OIDC configuration with OCM.
+Allows you to specify the mode to create your OIDC configuration. With the `manual` option, you receive AWS commands that setup the OIDC configuration within an S3 bucket. This option stores the private key in the Secrets Manager. With the `manual` option, the OIDC Endpoint URL is the URL for the S3 bucket. You must retrieve the Secrets Manager ARN to register the OIDC configuration with OpenShift Cluster Manager.
 
-Using the `auto` option, you receive the same OIDC configuration and AWS resources as the `manual` mode. One change is that ROSA calls AWS, so you do not need to do anything else. The OIDC Endpoint URL is the URL for the S3 bucket. The CLI retrieves the Secrets Manager ARN, registers the OIDC configuration with OCM, and reports the second `rosa` command that the user can run to continue with the creation of the STS cluster.
+Using the `auto` option, you receive the same OIDC configuration and AWS resources as the `manual` mode. One change is that ROSA calls AWS, so you do not need to do anything else. The OIDC Endpoint URL is the URL for the S3 bucket. The CLI retrieves the Secrets Manager ARN, registers the OIDC configuration with OpenShift Cluster Manager, and reports the second `rosa` command that the user can run to continue with the creation of the STS cluster.
```

Comment:
I don't love starting the sentence with a gerund, but I understand that you want to start by referencing the `auto` option. Suggestions/options, in no particular order of preference:
1. You receive the same OIDC configuration and AWS resources as the `manual` mode when using the `auto` option.
2. You receive the same OIDC configuration and AWS resources with the `auto` option as you do with the `manual` mode.
3. You receive the same OIDC configuration and AWS resources with both the `auto` option and the `manual` mode.
4. With both the `auto` option and the `manual` mode, you receive the same OIDC configuration and AWS resources. 
6. The `auto` option grants (or "gives you" or "gives the user") the same OIDC configuration and AWS resources as the `manual` mode. - I don't love this option because we attempt to avoid personification.

I might also change the sentence that immediately follows it ("One change is that ROSA...") because by mentioning "_one_ change", it suggests that the reader will at some point be alerted to other changes as well. Perhaps just "A significant difference between the two options is that when using the `auto` option, ROSA calls AWS, so you do not need to take any further actions." Or something of the sort.